js/script.js

```diff
@@ -0,0 +1,8 @@
+const navbar = document.querySelector(".navbar");
+const menuBtn = document.querySelector(".menu-btn"); 
+
+menuBtn.addEventListener ("click", (e)=> {
+  navbar.classList.toggle("navbar-active");  
+  menuBtn.firstElementChild.classList.toggle("none");
+  menuBtn.lastElementChild.classList.toggle("none");
+})
```

Comment:
Hi @danyhoshi, great job so far!
This file is not necessary, in this milestone don't needed a menu, just the html and css